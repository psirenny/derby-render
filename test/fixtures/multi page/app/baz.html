<Title:>
  baz

<Body:>
  <p>qux</p>
