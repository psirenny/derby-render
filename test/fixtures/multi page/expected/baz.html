<!DOCTYPE html><meta charset="utf-8"><title>baz</title><body><p>qux</p>
